blueprints/initializer/files/__root__/initializers/__name__.js

```diff
@@ -2,7 +2,4 @@ export function initialize(/* application */) {
   // application.inject('route', 'foo', 'service:foo');
 }
 
-export default {
-  name: '<%= dasherizedModuleName %>',
-  initialize
-};
+export default { initialize };
```

Comment:
Can we keep this multi-line to minimize diffs (here and for users, especially when adding before/afters)?